### PR TITLE
Zlib windows fix.

### PR DESCRIPTION
**Pull request by hmacdope**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -56,17 +57,32 @@ def extensions():
...
-            'pytng.pytng',
-            sources=glob('pytng/src/compression/*.c') + glob(
-                'pytng/src/lib/*.c') + glob('pytng/src/external/*.c') + ['pytng/pytng.pyx'],
-            include_dirs=[
-                "pytng/include/", "pytng/src/external/", "{}/include".format(sys.prefix),
-                np.get_include()
-            ],
-            library_dirs=["{}/lib".format(sys.prefix)],
-            libraries=['z'], ))
+    if sys.platform.startswith("win"):
```
Comment (richardjgowers) [inline setup.py:60]: Could we have the Extension defined once and the differences to DEF and libraries put behind the startswith condition?

Comment (hmacdope) [reply setup.py:60]: probably... do know anything about windows... I'm a bit stumped here.

**Comment by hmacdope** [comment]
Tagging @IAlibay here as I am honestly stumped, do you know if this is the proper way to link against a conda-installed library on Windows?

**Comment by hmacdope** [comment]
This PR has moved to #87. As it was accidentally from my master branch.

**Pull request closed without merging**